ui/backend/client_ui.py

```diff
@@ -97,3 +101,26 @@ def check_repo(self, repo_url, git_token=None, local_repo=False):
             except GitCommandError:
                 return False, 'GitCommandError'
         return True, None
+
+    def update_similar_snippets(self,
+                                target_snippet,
+                                state,
+                                repo_url,
+                                file_name=None,
+                                threshold=0.96):
+        discoveries = self.get_discoveries(repo_url,
+                                           file_name,
+                                           state_filter='new')[1]
```

Comment:
why do you only keep the [1] instead of the whole list?
are you sure you don't raise an IndexError if you don't have other discoveries?

Reply:
I added [1] because the get_discoveries method from the UI classes returns the number of discoveries as first element, which is useless in this case. I'm only retriving the actual discoveries, not their count. This is what adding [1] allows for.